### ISSUE TITLE
ADD ff2mpv button to Youtube Player controls

### ISSUE THREAD

**Issue by allanlaal** [woodruffw/ff2mpv#76]
**Is your feature request related to a problem? Please describe.**

ff2mpv currently takes up room in Firefox's Toolbar, but I only use it in Youtube 

**Describe the solution you'd like**

Youtube player  controls would have a ff2mpv button in them

**Describe alternatives you've considered**

tolerating it 

**Additional context**

![speed_buttons](https://user-images.githubusercontent.com/740826/154009413-d21f7e2e-44fb-4bcd-b660-a988e8fe85bf.png)

**Comment by woodruffw** [comment]
Thanks for the request. Could you describe how we'd implement this in the extension? Does it involve injecting detecting whether the page is YouTube and then injecting (more) JS into it?

**Comment by woodruffw** [comment]
(Also, note that the extension currently binds `Ctrl + Shift + V` to the "play in MPV" action. Does that work for you in terms of convenience?)

**Comment by allanlaal** [comment]
> (Also, note that the extension currently binds `Ctrl + Shift + V` to the "play in MPV" action. Does that work for you in terms of convenience?)

I only use my mouse for my "TV" computer, so a keyboard shortcut does not work

**Comment by allanlaal** [comment]
> Thanks for the request. Could you describe how we'd implement this in the extension? Does it involve injecting detecting whether the page is YouTube and then injecting (more) JS into it?

yes, commonly userscripts and plugins just inject those buttons into youtube controls. https://github.com/ajayyy/SponsorBlock would be an example of such a plugin

**Comment by woodruffw** [comment]
Hmm. I'm not opposed to this feature per se, but I'd like to see some more buy-in from other active users of ff2mpv (since it's not a feature that I'm likely to use). So I'll leave this open for a while and let people submit their opinions.

(A separate concern: I can see this being an arms race where we're always catching up to YouTube's frontend changes. I'd strongly prefer to avoid any changes that put us in that position, so this would have to be done in a way that avoids the need for constant new releases of ff2mpv.)

**Comment by benyaminl** [comment]
@woodruffw I think this is like "cat and mouse" game, so we need to kept updating it, that means we will often update with minor version when this is broken. Will it be okay? I'm okay to opening PR for this, but I will only open if it breaks, and I also not everyday accessing the youtube. Invidious is better and just right click the link open in MPV. 

I also have this kind of implementation on Google Drive sidebar to hide and show the sidebar, where I do frequently update the injected script (it's on greasyfork).

**Comment by woodruffw** [comment]
If you're willing to be the feature owner for these changes, then I'd happily accept a PR for this functionality. I'll probably end up pinging you if/when users report problems with it :slightly_smiling_face:

**Comment by allanlaal** [comment]
youtube doesn't scramble their generated class names each update like gcal and others
